.env.example

```
@@ -69,7 +69,7 @@ NEXT_PUBLIC_DD_SERVICE=noblocks
 NEXT_PUBLIC_DD_ENV=production
 NEXT_PUBLIC_DD_VERSION=
 NEXT_PUBLIC_DD_RUM_SAMPLE_RATE=100
-NEXT_PUBLIC_DD_SESSION_REPLAY_SAMPLE_RATE=0
+NEXT_PUBLIC_DD_SESSION_REPLAY_SAMPLE_RATE=100
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_DD_SESSION_REPLAY_SAMPLE_RATE key should go before the NEXT_PUBLIC_DD_SITE key

(UnorderedKey)